developer/src/kmcmpdll/DeprecationChecks.cpp

```diff
@@ -25,7 +25,7 @@ BOOL CheckForDeprecatedFeatures(PFILE_KEYBOARD fk) {
       // Keyman 7
       #define TSS_WINDOWSLANGUAGES 29
   */
-  int currentLineBackup = currentLine;
+  int oldCurrentLine = currentLine;
```

Comment:
Just making the backup variable name consistent -- same pattern used in three places. The currentLine change is made for error reporting.